src/components/AccountBox.js

```diff
@@ -181,7 +233,7 @@ function AccountBox({ currentAccount }) {
           trigger={<ActiveAccount address={address} type={type} />}
         />
       </CardBody>
-      <WalletBalances />
+      <WalletBalances hasActiveAccount={!!currentAccount} />
```

Comment:
Is it preferable to use the `useAccount()` hook in WalletBalances or just pass this as props? Maybe it's slightly more coupling for less computing overhead?

Reply:
I think its fine to pass the account as props here.